includes/admin/class-wcs-admin-post-types.php

```diff
@@ -1397,7 +1443,64 @@ public function handle_subscription_bulk_actions( $redirect_to, $action, $subscr
 			}
 		}
 
-		return esc_url_raw( add_query_arg( $sendback_args, $redirect_to ) );
+		return $sendback_args;
+	}
+
+	/**
+	 * Handles bulk trashing and deleting of subscriptions.
+	 *
+	 * @param array $ids Subscription IDs to be trashed or deleted.
+	 * @param bool  $force_delete When set, the subscription will be completed deleted. Otherwise, it will be trashed.
```

Comment:
```suggestion
	 * @param array $ids          Subscription IDs to be trashed or deleted.
	 * @param bool  $force_delete When set, the subscription will be completed deleted. Otherwise, it will be trashed.
```

Reply:
Just a minor WP coding standards thing. :)